libmamba/src/core/satisfiability_error.cpp

```diff
@@ -1642,11 +1642,11 @@ namespace mamba
                     {
                         if (tn.depth() == 1)
                         {
-                            write(" is requested and can be installed");
+                            write(" is requested and can not be installed");
```

Comment:
I'm not sure if this change is correct. but it looks like the proposed change is correct given the context of the code.